rir/src/utils/filesystem.h

```diff
@@ -0,0 +1,4 @@
+#pragma once
```

Comment:
a bit strange since we use the ifdef style everywhere else..

Reply:
Not everywhere

I do think we should standardize some of our style guide though: we use clang-format but there is also absolute/relative imports, method names, etc.